cmd/setup.go

```diff
@@ -1007,23 +1007,29 @@ func configureLoadpoints(conf globalconfig.All) error {
 
 		settings := coresettings.NewConfigSettingsAdapter(log, &conf)
 
-		dynamic, static, err := loadpoint.SplitConfig(cc.Other)
-		if err != nil {
-			return &DeviceError{cc.Name, err}
+		dynamic, static, e := loadpoint.SplitConfig(cc.Other)
```

Comment:
Ich verstehe die Änderung nicht, ggf. auch zuviel diff. können wir bitte beim üblich err für error bleiben?

Reply:
Das Problem ist hier, dass wir bislang beim ersten Fehler abbrechen. Das führt dazu, dass die LP Instanz nicht zurückgegeben und angemeldet wurde. Wenn es im UI bearbeitbare bleiben soll müssen wir das komplette Objekt aufbauen (inkl nachfolgender Werte wie Title, Prio) und es dann mit einem Fehler zurückliefen. Das fail and return fast Pattern funktioniert hier nicht. Glücklicherweise hängen die Felder aktuell nicht voneinander ab. Heißt für die nachfolgenden Aufrufe ist es heute nicht wichtig, ob der Charger gefunden und verbunden wurde.

Reply:
wenn der erste Schritt failed (nil)- dann ist es sehr mutig die weiteren auszuführen, denn die greifen dann ggf. auf etwas zu dass es nicht gibt. Oder übersehe ich da was?

Reply:
Bei `SplitConfig` können wir von mir aus auch sofort fallen. da kann/darf die Ursache eigentlich keine Nutzerinteraktion sein.